editor/editor_node.cpp

```diff
@@ -3702,6 +3702,18 @@ void EditorNode::fix_dependencies(const String &p_for_file) {
 
 int EditorNode::new_scene() {
 	int idx = editor_data.add_edited_scene(-1);
+	// Remove [empty] scene if any other scene is loaded.
```

Comment:
The [empty] is also removed if there is another [empty] (so no scene loaded), so the comment is not exactly correct. Maybe something like "Remove placeholder empty scene." would be better.

Reply:
one single commit , Done.
rewriting comment , Done.